src/Infrastructure/Web/Web.csproj

```diff
@@ -1,31 +1,40 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), common.props))\common.props" />
```

Comment:
This look like you used old version of the file during merge process. Please, return original lines from master.

Reply:
Check whole file, please.